service/src/main/kotlin/fi/espoo/evaka/messaging/bulletin/BulletinQueries.kt

```diff
@@ -126,28 +124,53 @@ fun Database.Transaction.sendBulletin(
     // language=sql
     val insertBulletinInstancesSql = """
         WITH children AS (
-            SELECT pl.child_id, br.id AS bulletin_receiver_id
+            SELECT child_id, bulletin_id FROM bulletin_receiver
+            WHERE
+            bulletin_id = :bulletinId 
+            AND unit_id IS NOT NULL 
```

Comment:
Based on kotlin data class it looks to me like bulletin_receiver.unit_id should never be null. If that is the case then lets remove these checks from here and add the missing database not null constraint instead.

Reply:
I guess this depends on how we want to handle area- and citywise bulletins on the UI level - do we want to handle eg. Espoon keskus as single recipient or a collection of units 🤷